src/editor/components/components/ActionBar/ActionBar.component.jsx

```diff
@@ -28,6 +29,19 @@ const ActionBar = ({ handleAddClick, isAddLayerPanelOpen }) => {
     setCursorEnabled(true);
   };
 
+  const handleCameraCLick = () => {
+    Events.emit('camera_clicked');
+    const screenshotEl = document.getElementById('screenshot');
+    screenshotEl.play();
+
+    posthog.capture('screenshot_taken', {
+      scene_id: STREET.utils.getCurrentSceneId()
+    });
+
+    screenshotEl.setAttribute('screentock', 'type', 'png');
```

Comment:
I think jpg would be a better default, png can really be heavy in size.

Reply:
png seems to be the better default for graphics design / presentation purposes. it also seems to be used a bit more than jpg in our usage.